src/main/java/org/opensearch/eval/App.java

```diff
@@ -87,7 +87,7 @@ public static void main(String[] args) throws Exception {
             if(CoecClickModel.CLICK_MODEL_NAME.equalsIgnoreCase(clickModelType)) {
 
                 // Create the judgments index if it does not already exist.
-                searchEngine.createJudgmentsIndex();
+                searchEngine.createIndex(Constants.JUDGMENTS_INDEX_NAME, Constants.JUDGMENTS_INDEX_MAPPING);
```

Comment:
needs a check if the index already exists first. Throws an error if index already present.
Something like this:
```
if(!searchEngine.doesIndexExist(Constants.JUDGMENTS_INDEX_NAME)) {
                    searchEngine.createIndex(Constants.JUDGMENTS_INDEX_NAME, Constants.JUDGMENTS_INDEX_MAPPING);
}
```